include/materials/material_helper.hpp

```diff
@@ -199,6 +199,18 @@ class MaterialHelper {
     return Emt_;
   }
 
+  double Eh(double E) {
```

Comment:
```suggestion
  double heating(double E) {
```
This isn't exactly a cross section, so I think we should change the name in this way.